rc/extra/autorestore.kak

```diff
@@ -44,19 +48,20 @@ def autorestore-restore-buffer -docstring "Restore the backup for the current fi
 
 ## Remove all the backups that have been created for the current buffer
 def autorestore-purge-backups -docstring "Remove all the backups of the current buffer" %{
-    nop %sh{
+    %sh{
         if [ ! -f "${kak_buffile}" ]; then
             exit
         fi
 
         buffer_basename="${kak_bufname##*/}"
         buffer_dirname=$(dirname "${kak_bufname}")
+        log=$(rm -v "${buffer_dirname}/.${buffer_basename}.kak."* 2>&1)
 
-        ls -1 "${buffer_dirname}"/."${buffer_basename}".kak.* 2>/dev/null | while read -r f; do
-            rm -f "${f}"
-        done
+        printf %s\\n "
+          echo -markup \"{Information}Backup files removed\"
+          echo -debug \"Removed backup files: ${log}\"
```

Comment:
Protip: the `echo` command will join all of its arguments with a single space, so you don't need to escape the message, e.g. `echo -debug Removed backup files: ${log}` (no quotes needed).

Reply:
Nice. But can I write, e.g., `echo -debug foo \n bar` ?